python/cuml/cuml/dask/ensemble/base.py

```diff
@@ -217,14 +220,13 @@ def _partial_inference(self, X, op_type, delayed, **kwargs):
                     pure=False,
                 )
             )
-        shape = (X.shape[0], 1, self.num_classes)
+
+        meta = cp.zeros((0,) * 3, dtype=cp.float32)
         objs = [
-            dask.array.from_delayed(partial_inf, shape=shape, dtype=np.float32)
+            dask.array.from_delayed(partial_inf, shape=shape, meta=meta)
             for partial_inf in partial_infs
         ]
-        result = dask.array.concatenate(
-            objs, axis=1, allow_unknown_chunksizes=True
```

Comment:
I believe `allow_unknown_chunksizes=True` is unnecessary here. We're concatenating along `axis=1`, which will always be known.